backend-rust/src/indexer.rs

```diff
@@ -707,6 +707,8 @@ async fn save_genesis_data(endpoint: v2::Endpoint, pool: &PgPool) -> anyhow::Res
     let mut tx = pool.begin().await.context("Failed to create SQL transaction")?;
     let genesis_height = v2::BlockIdentifier::AbsoluteHeight(0.into());
     {
+        let epoch_duration = client.get_consensus_info().await?.epoch_duration.num_milliseconds();
```

Comment:
As per talk I do believe this could be stored inmem on the api upon startup. Maybe even just hardcode the values per env.

If you find that too annoying just keep it as is, since this will not be performance impacting at all.

Reply:
Based on some discussion. We do not want `envs` because they need to be set at startup `correctly` and locked-up for each network as such.

To look up the value at start-up in the `graphQL_API` service is a good suggestions and we will consider adding a node client to the service if we have more such cases. 

For now we keep this look-up in the indexer.